modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/resources/META-INF/resources/notification/form_entry_add_body.ftl

```diff
@@ -27,7 +27,7 @@
 		.field-value {
 			font-size: 16px;
 			margin: 0;
-			white-space: pre-line;
+			white-space: pre;
```

Comment:
@rijema, do we really need this CSS property or the tag `<pre>` is enough to attend our needs?

Reply:
Hi @natocesarrego , I also tested only with the tag pre and yes, it solves both use cases (the one whose I solved before using the CSS attribute and this one specifically to the Outlook). I was just double checking with the CSS change. Do you want me to update this PR with only the tag change?

Reply:
Yes, please, @rijema. :slightly_smiling_face: